app/views/vocab_sheets/_vocab_sheet_item.html.haml

```diff
@@ -5,9 +5,9 @@
       = link_to sign_url(vocab_sheet_item.sign_id), { class: "drawing vocab_sheet_drawing drawing vocab_sheet_drawing__size--#{size}" } do
         = image_tag "signs/#{vocab_sheet_image_width}-#{vocab_sheet_image_height}/#{vocab_sheet_item.drawing}"
       = form_tag vocab_sheet_item_path(vocab_sheet_item), method: :put do
-        = text_area_tag "item[name]", vocab_sheet_item.name, class: 'item-name vocab-sheet__text-input h3', id: "item_#{vocab_sheet_item.id}_name"
+        = text_area_tag "item[name]", vocab_sheet_item.name, class: 'item-name vocab-sheet__text-input h3', id: "item_#{vocab_sheet_item.id}_name", maxLength:vocab_sheet_max_field_length
```

Comment:
does that maxLength work? It doesn't seem to be mentioned in the docs . It's on `text_field_tag` but not on `text_area_tag`.

https://api.rubyonrails.org/v5.1/classes/ActionView/Helpers/FormTagHelper.html#method-i-text_area_tag

Reply:
The max length isn't doing anything afaict. It doesn't limit in firefox or chrome. I can't see that 100 turning up in the mark up either.

Reply:
https://stackoverflow.com/questions/886352/maximum-length-on-a-textarea-in-ruby-on-rails